AssertionError: Expected number of dimensions to be 2, got 4

**Describe the bug**
IS, FID, GS, KID, MSID and PR don't work, because of `AssertionError: Expected number of dimensions to be 2, got 4`. 

**To Reproduce**
Copy paste example code from [the documentation](https://piq.readthedocs.io/en/latest/modules.html#feature-metrics), for example:
```
from piq import FID
import torch
loss = FID()
x = torch.rand(3, 3, 256, 256, requires_grad=True)
y = torch.rand(3, 3, 256, 256)
output = loss(x, y)
output.backward()
```

**Expected behavior**
This should run without errors, but instead, this results in: 
```
---------------------------------------------------------------------------

AssertionError                            Traceback (most recent call last)

<ipython-input-20-9300c877d44d> in <module>()
      4 x = torch.rand(3, 3, 256, 256, requires_grad=True)
      5 y = torch.rand(3, 3, 256, 256)
----> 6 output = loss(x, y)
      7 output.backward()

/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py in _call_impl(self, *input, **kwargs)
   1049         if not (self._backward_hooks or self._forward_hooks or self._forward_pre_hooks or _global_backward_hooks
   1050                 or _global_forward_hooks or _global_forward_pre_hooks):
-> 1051             return forward_call(*input, **kwargs)
   1052         # Do not call functions when jit is used
   1053         full_backward_hooks, non_full_backward_hooks = [], []

/usr/local/lib/python3.7/dist-packages/piq/base.py in forward(self, x_features, y_features)
     13 
     14     def forward(self, x_features: torch.Tensor, y_features: torch.Tensor) -> torch.Tensor:
---> 15         return self.compute_metric(x_features, y_features)
     16 
     17     @torch.no_grad()

/usr/local/lib/python3.7/dist-packages/piq/fid.py in compute_metric(self, x_features, y_features)
    169             The Frechet Distance.
    170         """
--> 171         _validate_input([x_features, y_features], dim_range=(2, 2), size_range=(1, 2))
    172         # GPU -> CPU
    173         mu_x, sigma_x = _compute_statistics(x_features.detach().to(dtype=torch.float64))

/usr/local/lib/python3.7/dist-packages/piq/utils/common.py in _validate_input(tensors, dim_range, data_range, size_range)
     34 
     35         if dim_range[0] == dim_range[1]:
---> 36             assert t.dim() == dim_range[0], f'Expected number of dimensions to be {dim_range[0]}, got {t.dim()}'
     37         elif dim_range[0] < dim_range[1]:
     38             assert dim_range[0] <= t.dim() <= dim_range[1], \

AssertionError: Expected number of dimensions to be 2, got 4
```

--- a/piq/fid.py
+++ b/piq/fid.py
@@ -142,11 +142,10 @@ class FID(BaseFeatureMetric):
     But dimensionalities should match, otherwise it won't be possible to correctly compute statistics.
 
     Examples:
-        >>> loss = FID()
-        >>> x = torch.rand(3, 3, 256, 256, requires_grad=True)
-        >>> y = torch.rand(3, 3, 256, 256)
-        >>> output = loss(x, y)
-        >>> output.backward()
+        >>> fid_metric = FID()
+        >>> x_feats = torch.rand(10000, 1024)
+        >>> y_feats = torch.rand(10000, 1024)
+        >>> fid: torch.Tensor = fid_metric(x_feats, y_feats)
 
     References:
         Heusel M. et al. (2017).

--- a/piq/gs.py
+++ b/piq/gs.py
@@ -158,11 +158,10 @@ class GS(BaseFeatureMetric):
         num_workers: Number of proccess used for GS computation.
 
     Examples:
-        >>> loss = GS()
-        >>> x = torch.rand(3, 3, 256, 256, requires_grad=True)
-        >>> y = torch.rand(3, 3, 256, 256)
-        >>> output = loss(x, y)
-        >>> output.backward()
+        >>> gs_metric = GS()
+        >>> x_feats = torch.rand(10000, 1024)
+        >>> y_feats = torch.rand(10000, 1024)
+        >>> gs: torch.Tensor = gs_metric(x_feats, y_feats)
 
     References:
         Khrulkov V., Oseledets I. (2018).

--- a/piq/isc.py
+++ b/piq/isc.py
@@ -75,11 +75,10 @@ class IS(BaseFeatureMetric):
         distance: How to measure distance between scores: ``'l1'`` | ``'l2'``. Default: ``'l1'``.
 
     Examples:
-        >>> loss = IS()
-        >>> x = torch.rand(3, 3, 256, 256, requires_grad=True)
-        >>> y = torch.rand(3, 3, 256, 256)
-        >>> output = loss(x, y)
-        >>> output.backward()
+        >>> is_metric = IS()
+        >>> x_feats = torch.rand(10000, 1024)
+        >>> y_feats = torch.rand(10000, 1024)
+        >>> is: torch.Tensor = is_metric(x_feats, y_feats)
 
     References:
         "A Note on the Inception Score" https://arxiv.org/pdf/1801.01973.pdf

--- a/piq/kid.py
+++ b/piq/kid.py
@@ -154,11 +154,10 @@ class KID(BaseFeatureMetric):
             This function will return ``Tuple[torch.Tensor, torch.Tensor]`` in this case. Default: False
 
     Examples:
-        >>> loss = KID()
-        >>> x = torch.rand(3, 3, 256, 256, requires_grad=True)
-        >>> y = torch.rand(3, 3, 256, 256)
-        >>> output = loss(x, y)
-        >>> output.backward()
+        >>> kid_metric = KID()
+        >>> x_feats = torch.rand(10000, 1024)
+        >>> y_feats = torch.rand(10000, 1024)
+        >>> kid: torch.Tensor = kid_metric(x_feats, y_feats)
 
     References:
         Demystifying MMD GANs https://arxiv.org/abs/1801.01401

--- a/piq/msid.py
+++ b/piq/msid.py
@@ -321,11 +321,10 @@ class MSID(BaseFeatureMetric):
             ``'max'`` to find the maximum absolute difference between two descriptors over temperature
 
     Examples:
-        >>> loss = MSID()
-        >>> x = torch.rand(3, 3, 256, 256, requires_grad=True)
-        >>> y = torch.rand(3, 3, 256, 256)
-        >>> output = loss(x, y)
-        >>> output.backward()
+        >>> msid_metric = MSID()
+        >>> x_feats = torch.rand(10000, 1024)
+        >>> y_feats = torch.rand(10000, 1024)
+        >>> msid: torch.Tensor = msid_metric(x_feats, y_feats)
 
     References:
         Tsitsulin, A., Munkhoeva, M., Mottin, D., Karras, P., Bronstein, A., Oseledets, I., & Müller, E. (2019).

--- a/piq/pr.py
+++ b/piq/pr.py
@@ -61,10 +61,10 @@ class PR(BaseFeatureMetric):
         nearest_k: Nearest neighbor to compute the non-parametric representation. Shape :math:`1`
         
     Examples:
-        >>> loss = PR()
-        >>> x = torch.rand(3, 3, 256, 256, requires_grad=True)
-        >>> y = torch.rand(3, 3, 256, 256)
-        >>> precision, recall = loss(x, y)
+        >>> pr_metric = PR()
+        >>> x_feats = torch.rand(10000, 1024)
+        >>> y_feats = torch.rand(10000, 1024)
+        >>> precision, recall = pr_metric(x_feats, y_feats)
 
     References:
         Kynkäänniemi T. et al. (2019).

